data/test_endpoints/event_brokers/kafka_topics.yml

```diff
@@ -0,0 +1,9 @@
+event_broker:
+  type: kafka
+  security_protocol: PLAINTEXT
+  topic: topic
```

Comment:
Just a personal note:
I now came to a conclusion that we should update our policy to enable us to mark certain fields for deprecation. This will give us more freedom to do feature augmentation like in this example, where we are augmenting the number of topics to which events can be pushed.